### PR TITLE
Add An Unexpected Company

### PR DESCRIPTION
**Pull request by Sabre142**
Related to #653.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -6935,6 +6935,33 @@ plugins:
...
+        name: 'An Unexpected Company'
+  - name: 'An Unexpected Company.esp'
+    req: [ *SilentVoice ]
+    inc:
+      - 'A Family Problem.esp'
+      - 'LGVampire.esp'
+    msg:
+      - <<: *obsolete
+        subs: [ '[An Unexpected Company v1-2-2 PATCH (with increased follow distance)](https://www.nexusmods.com/oblivion/mods/41801/)' ]
+        condition: 'checksum("An Unexpected Company.esp", 9E6F5833)'
```
Comment (sibir-ine) [inline masterlist.yaml]: The plugin's version number can be used.

```suggestion
        condition: 'version("An Unexpected Company.esp", "1.2.2", <)'
```

**Review by sibir-ine — DISMISSED** [review]

**Comment by Sabre142** [comment]
Wrye Bash 309.1 can't recognize the version info in this mod's description.

File: masterlist.yaml [hunk 2]
```diff
@@ -6935,6 +6935,33 @@ plugins:
...
+        condition: 'version("An Unexpected Company.esp", "1.2.2", <)'
+      - <<: *patchIncluded
+        subs: [ 'Better Cities' ]
+        condition: 'active("Better Cities Resources.esm") and not file("An Unexpected Company - Better Cities.esp")'
+      - *versionXofYorGreaterRequiredOBSEv20
+    tag: [ Graphics ]
+    clean:
+      - crc: 0xD5C1D187
+        util: 'TES4Edit v4.0.4b'
+  - name: 'An Unexpected Company - Better Cities.esp'
```
Comment (MacSplody) [inline masterlist.yaml:6987]: Some rules to match suggested order.

image

```suggestion
  - name: 'An Unexpected Company - Better Cities.esp'
    after: [ 'Better Cities Full.esp' ]
    req:
      - name: 'Better Cities SKINGRAD.esp'
        display: 'Better Cities Full.esp or Better Cities SKINGRAD.esp'
        condition: 'not file("Better Cities Full.esp")'
```

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**